wordpress-mobile/WordPress-iOS
# [Insights Management] Resize subtitles after adding stat

Fixes #12709 

To test:
- Go to Stats > Insights.
- Remove _Tags and Categories_ and _Publicize_ if necessary.
- Add both back to Insights.
- Verify the subtitles display properly

NOTE: Subtitles are only shown when there is data, so use a site that has data for those stats. en.blog usually does.

![subtitles](https://user-images.githubusercontent.com/1816888/66961600-49dda300-f02c-11e9-857f-1bc7e2ef0832.png)

Update release notes:
